app/views/directives/pipeline-status.html

```diff
@@ -0,0 +1,34 @@
+<span class="pipeline-status" ng-class="status">
+  <span ng-switch="status" class="hide-ng-leave">
+    <!-- See https://github.com/jenkinsci/pipeline-stage-view-plugin/blob/master/rest-api/src/main/java/com/cloudbees/workflow/rest/external/StatusExt.java -->
+    <span ng-switch-when="ABORTED">
+      <i class="fa fa-ban text-muted" aria-hidden="true"></i>
+      <span class="sr-only">Aborted</span>
+    </span>
+    <span ng-switch-when="FAILED">
+      <span class="fa-stack">
+        <i class="fa fa-circle fa-stack-2x"></i>
+        <i class="fa fa-times fa-stack-1x fa-inverse"></i>
+      </span>
+      <span class="sr-only">Failed</span>
+    </span>
+    <span ng-switch-when="IN_PROGRESS">
+      <i class="fa fa-refresh fa-spin" aria-hidden="true"></i>
+      <span class="sr-only">In Progress</span>
+    </span>
+    <span ng-switch-when="NOT_EXECUTED">
+      <!-- No icon -->
+    </span>
+    <span ng-switch-when="PAUSED_PENDING_INPUT">
+      <i class="fa fa-pause" aria-hidden="true"></i>
+      <span class="sr-only">Paused Pending Input</span>
+    </span>
+    <span ng-switch-when="SUCCESS">
```

Comment:
Thats fine, just want to make sure something is there to catch this case as
opposed to blank space.

On Thu, May 19, 2016 at 1:15 PM, Sam Padgett <notifications@github.com>
wrote:

> In app/views/directives/pipeline-status.html
> <https://github.com/openshift/origin-web-console/pull/20#discussion_r63918201>
> :
>
> > +        <i class="fa fa-times fa-stack-1x fa-inverse"></i>
> > +      </span>
> > +      <span class="sr-only">Failed</span>
> > +    </span>
> > +    <span ng-switch-when="IN_PROGRESS">
> > +      <i class="fa fa-refresh fa-spin" aria-hidden="true"></i>
> > +      <span class="sr-only">In Progress</span>
> > +    </span>
> > +    <span ng-switch-when="NOT_EXECUTED">
> > +      <!-- No icon -->
> > +    </span>
> > +    <span ng-switch-when="PAUSED_PENDING_INPUT">
> > +      <i class="fa fa-pause" aria-hidden="true"></i>
> > +      <span class="sr-only">Paused Pending Input</span>
> > +    </span>
> > +    <span ng-switch-when="SUCCESS">
>
> I show a horizontal gray line with no icon for not executed. I could do
> the same for unknown here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/20/files/b58a5686ba5fd1d681075d46d6f94b53d56a872b#r63918201>
>